tests/kubernetes/utils.py

```diff
@@ -78,12 +113,11 @@ def create_serviceaccount(body=None, namespace="default", timeout=K8S_API_TIMEOU
 
 def has_configmap(name, namespace="default"):
     api = kube_client.CoreV1Api()
-    configmaps = api.list_namespaced_config_map(namespace=namespace)
-    if configmaps:
-        for cm in configmaps.items:
-            if cm.metadata.name == name:
-                return True
-    return False
+    try:
+        api.read_namespaced_config_map(name, namespace=namespace)
+        return True
+    except:
```

Comment:
This catch-all exception handler (and the others like it) will mask other errors in the K8s API, which could make debugging test failures a lot more difficult.  Try and figure out exactly what exception the K8s client throws when the resource does not exist and catch it only.

Reply:
Let any other types of exceptions bubble up and crash the test runner so it is obvious if there are problems.